programs/drift/src/controller/orders.rs

```diff
@@ -250,11 +250,15 @@ pub fn place_perp_order(
         "must be perp order"
     )?;
 
+    let mut order_slot: u64 = slot;
+    if let Some(swift_taker_order_slot) = options.swift_taker_order_slot {
```

Comment:
nit: think we can move this to a method on the PlaceOrderOptions

Reply:
done